### PR TITLE
short link for k8s cheat sheet

### LINKED ISSUE #23909: Short link for kubectl cheat sheet
**This is a Feature Request**

**What would you like to be added**
Add a short link for the [kubectl cheat sheet](https://k8s.io/docs/reference/kubectl/cheatsheet/).

**Why is this needed**
A short link, eg https://k8s.io/cheatsheet, is easier to type and remember.

**Comments**
I recommend **not** using the usual 301 redirect here because it's not a permanent move. Instead, use a 307 redirect.

- Should we repeat this for localizations? Eg https://k8s.io/de/spickzettel

In the cheat sheet page itself, consider mentioning the short link in the text of the page.
/kind feature
/language en

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Close #23909

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -369,7 +369,7 @@
 #/docs/user-guide/kubectl/v1.9/*     https://v1-9.docs.kubernetes.io/docs/reference/generated/kubectl/kubectl-commands/:splat 301
 #/docs/user-guide/kubectl/v1.10/*     /docs/reference/generated/kubectl/kubectl-commands/:splat 301
 /docs/user-guide/kubectl-conventions/     /docs/reference/kubectl/conventions/
-/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/
+/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/ 307
```
Comment (kbhawkey) [inline static/_redirects:372]: Hi @neha-viswanathan ,
Would you check the netlify documentation. I don't think `307` is supported.

**Comment by neha-viswanathan** [comment]
@sftim @kbhawkey the issue #23909 mentions not to use `301` since this is not a permanent redirect. So, I have used `302`.  Please review. Thanks!

**Comment by sftim** [comment]
Hi @neha-viswanathan 

https://k8s.io/docs/user-guide/kubectl-cheatsheet/ is a bit long for a short link. How about https://k8s.io/cheatsheet ?

Would you like advice on how to implement that?

**Comment by neha-viswanathan** [comment]
@sftim Please correct me if I'm wrong- Thanks
```/cheatsheet/     /docs/reference/kubectl/cheatsheet/ 302```

**Comment by sftim** [comment]
> `/cheatsheet/ /docs/reference/kubectl/cheatsheet/ 302`

I would *also* redirect:
`/cheatsheet /docs/reference/kubectl/cheatsheet/ 302`

File: static/_redirects [hunk 2]
```diff
@@ -367,7 +367,7 @@
 /docs/user-guide/kubectl-overview/     /docs/reference/kubectl/overview/
 /docs/user-guide/kubectl/     /docs/reference/generated/kubectl/kubectl-options/
 /docs/user-guide/kubectl-conventions/     /docs/reference/kubectl/conventions/
-/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/
```
Comment (sftim) [inline static/_redirects:370]: It's OK to leave this line in. I don't think it's a good idea to remove it. Instead, I recommend keeping line 370 from the original and then _adding_ an extra redirect.

**Review by sftim — COMMENTED** [review]
/lgtm
/hold

I'd like to run this past SIG Docs - maybe a weekly meeting? - to check that people are happy with the approach.

**Comment by sftim** [comment]
Discussed in SIG Docs weekly meeting, good to go
/hold cancel

**Comment by annajung** [comment]
discussed in Sig Docs meeting Jan 12th, no objections from the team
/approve

**Pull request merged**